android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java

```diff
@@ -101,7 +117,38 @@ protected void onResume() {
         realmRepo.getSessionsByLocation(location)
                 .addChangeListener((sessions, orderedCollectionChangeSet) -> {
                     mSessions.clear();
+                    String upcomingTitle = "";
+                    String track = "";
+                    String color = null;
```

Comment:
Okay, thanks for the clarification. Did the changes. Please review it once again.